pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -690,15 +690,14 @@ func (m *MultishareOpsManager) listMatchedInstances(ctx context.Context, req *cs
 // 7, Both source and target instance should have the same connect mode.
 // 8. Both source and target instance should have the same KmsKeyName.
 func isMatchedInstance(source, target *file.MultishareInstance, req *csi.CreateVolumeRequest) (bool, error) {
```

Comment:
please update the comment above the function "Both source and target instance..."

Reply:
done.